### PR TITLE
update display

### PR DESCRIPTION
**Pull request by Mxdcq**

### CODE REVIEW

File: app/js/event/event.filter.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    .filter('distanceFormat', distanceFormat);
+
+  distanceFormat.$inject = [];
+
+  function distanceFormat() {
+
+    return function(x) {
+      var distance;
+      if(isNaN(x)){
+        return;
```
Comment (antti-TIEA4) [inline app/js/event/event.filter.js]: The filter should return the original value if it is not a number.
In other words, line 15 should be
```
        return x;
```

**Comment by Mxdcq** [comment]
I am waiting you to merge it. Hope there is no mistake.

**Comment by antti-TIEA4** [comment]
I made a comment above:
> The filter should return the original value if it is not a number.
> In other words, line 15 should be
> ```
>         return x;
> ```

**Comment by antti-TIEA4** [comment]
Sorry, I made a comment but forgot to submit it and then Chrome updated and died.

File: app/js/event/event.filter.js (hunk 1 above)
Comment (Mxdcq) [inline app/js/event/event.filter.js]: Done

**Pull request merged**